georg-ui/src/components/Detail.vue

```diff
@@ -79,57 +84,71 @@
         </v-list-item-content>
       </v-list-item>
       <v-divider v-if="!isNewMarker" inset></v-divider>
-      <v-list-item v-if="!isNewMarker">
-        <v-list-item-icon>
-          <v-icon color="blue darken-2">mdi-database-import</v-icon>
-        </v-list-item-icon>
-        <v-list-item-content>
-          <v-list-item-title>Data från {{ dataFromSource }}</v-list-item-title>
-        </v-list-item-content>
-        <v-btn
-          v-if="!isGbif"
-          icon
-          :href="datasourcelink"
-          target="_blank"
-          id="wofLink"
+      <v-list-item-group>
+        <v-list-item
+          v-if="!isNewMarker"
+          v-on="
+            isGbif ? { click: openOrCloseGbifData } : { click: openWofLink }
```

Comment:
This might not working. This link not only to wof, but also for svh.

Reply:
Should be fixed in https://github.com/Naturhistoriska/Georg/pull/42/commits/2baf90edf90d8a10ba9f216156cbfdb9d3949e7c